packages/gatsby-plugin-preact/README.md

```diff
@@ -10,7 +10,7 @@ React.
 
 ## Install
 
-`npm install --save gatsby-plugin-preact preact@beta preact-compat`
```

Comment:
Since PreactX have it's own `preact/compat`. It unnecessary to add `preact-compat` into package.